src/python/scripts/run_gerp.py

```diff
@@ -28,45 +28,78 @@
     $ python run_gerp.py --msa_file alignment.mfa --tree_file tree.nw --gerp_exe_dir path/to/gerp/
 
 """
-
+import argparse
+import json
 import os
 import subprocess
 
-import argschema
+def add_ce_to_json(ce_file: str, json_file: str) -> None:
+    """Enrich the json file with the constrained elements.
 
+    Args:
+        ce_file: file containing the constrained elements (csv format).
+        json_file: json file with genome coordinate level.
 
-class InputSchema(argschema.ArgSchema):
-    """Calculates the Genomic Evolutionary Rate Profiling (GERP) of a multiple sequence alignment (MSA)."""
+    """
+    with open(json_file, 'r') as json_file_obj:
+        align_set = json.load(json_file_obj)
 
-    msa_file = argschema.fields.InputFile(
-        required=True, description="MSA file (MFA format)"
-    )
-    tree_file = argschema.fields.InputFile(
-        required=True, description="Tree file (Newick format). Must include every species in the MSA."
-    )
-    depth_threshold = argschema.fields.Float(
-        required=False,
-        description="Constrained elements' depth threshold for shallow columns, in substitutions "
-                    "per site. By default, 0.5."
-    )
-    gerp_exe_dir = argschema.fields.InputDir(
-        required=False,
-        description="Path where 'gerpcol' and 'gerpelem' binaries can be found. By default, resort to $PATH."
-    )
+    constrained_elems = []
+    with open(ce_file) as ce_file_obj:
+        # enrich json file with constrained elements info
+        for cons_elem in ce_file_obj:
+            if cons_elem.rstrip() == "":
+                continue
```

Comment:
Is it a frequent occurrence for a constrained element file to contain empty lines? Would an empty line be output during a normal run, or could it potentially be a sign that something has gone wrong?